docs/guides/dns-over-https.md

```diff
@@ -57,7 +57,11 @@ Create a `cloudflared` user to run the daemon:
 sudo useradd -s /usr/sbin/nologin -r -M cloudflared
 ```
 
-Proceed to create a configuration file for `cloudflared` by copying the following in to `/etc/default/cloudflared`. This file contains the command-line options that get passed to cloudflared on startup:
+Proceed to create a configuration file for `cloudflared`:
+```bash
+sudo nano /etc/default/cloudflared
+```
+Edit configuration file by copying the following in to `/etc/default/cloudflared`. This file contains the command-line options that get passed to cloudflared on startup:
```

Comment:
docs/guides/dns-over-https.md:61 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```bash"]
docs/guides/dns-over-https.md:63 MD031/blanks-around-fences Fenced code blocks should be surrounded by blank lines [Context: "```"]

```suggestion
```bash
sudo nano /etc/default/cloudflared
```

Edit configuration file by copying the following in to `/etc/default/cloudflared`. This file contains the command-line options that get passed to cloudflared on startup:
```